sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/projectoverview/Project.groovy

```diff
@@ -0,0 +1,63 @@
+package life.qbic.business.projectoverview
+
+import life.qbic.datamodel.samples.Sample
+
+/**
+ * <h1><short description></h1>
+ *
+ * <p><detailed description></p>
+ *
+ * @since 1.0.0
+ *
+*/
+class Project {
+
+    final String projectCode
+    final String projectDescription
+    final List<Sample> samples
+    float progress
+    FailedSamplesRatio failedSamples
```

Comment:
Is this a Ratio or a number indicating the amount of failedSamples? 
```suggestion
    FailedSamplesRatio failedSamplesRatio
```

Reply:
Check the screenshot. It's both, the ratio and the number indicating the failed samples. Since it mainly is about the number of failed samples rather than the ratio (which I introduced to put the number of failed samples into perspective) I would keep it as is.